roles/etcd/handlers/main.yml

```diff
@@ -6,21 +6,14 @@
     - reload etcd
     - wait for etcd up
 
-- name: restart etcd-proxy
```

Comment:
It's not compatible with TLS. I can't get it to actually start up without crashing and thinking it's _not_ a proxy.